documentation/IDTA-01002-3/modules/ROOT/pages/http-rest-api/http-rest-api.adoc

```diff
@@ -373,7 +373,19 @@ path-suffix= $metadata/$value/$reference/$path or no suffix for normal
 ?extent=WithoutBLOBValue/WithBLOBValue
 
 |PutSubmodel |PUT |/submodel |
-|PatchSubmodel |PATCH |/submodel |path-suffix=$metadata/$value or no path for normal
+|PatchSubmodel |PATCH |/submodel |path-suffix=/$metadata or /$value 
+
+For /$metadata: +
+The serialised submodel in the request body must not contain any SubmodelElements. +
+The server must overwrite all metadata-relevant Submodel fields with the received fields. This means in particular that previously used fields of the Submodel instance that are missing in the request body indicate a deletion of them. It is not possible to change single items in contained arrays. In particular, a client must therefore send the complete content of arrays (e.g. for `description` or `displayName`).
+
+For /$value: +
+The server must overwrite all ValueOnly-relevant Submodel fields with the received values. This means in particular that previously used fields of the Submodel instance that are missing in the request body indicate a deletion of them. It is not possible to change single items in contained arrays, for instance, for SubmodelElementLists.
```

Comment:
This seems to be a 1:1 copy from other sections but I think it would be better to write something different and more specific to modifying the value of a submodel. E.g., this text mentions SubmodelElementLists which are not relevant here because we are talking about modifying an array of SubmodelElements without the need to know what the elements are.

More important, this won't work becaue valueOnly serialization does not contain all information to "create" new elements. Imaging a submodel without any child elements. Through PATCH on /$value we now send something like `{"foo": 42}`. What is the result? We could deduct that this is a property called foo, but we cannot infer the type of foo (double, int, short, ???) as the valueOnly serialization does not hold the necessary type information. 
As a consequence, adding the elements via valueOnly must not be allowed which would require quite complex processing to check because of arbitrary nesting depth and also because type checking would need to happen.

Reply:
> but I think it would be better to write something different

Always open for better proposals

Reply:
> More important, this won't work because valueOnly serialization does not contain all information to "create" new elements.

It was the outcome of the last group call to define it like that. Please raise the topic again there, together with a respective formulation, to cover also this case.

Reply:
This is not an issue of how to phrase things but a technical issue that this approach will not work. 
In the REST API WG calls I did propose to follow standard JSON Merge Patch behavior and drop support for PATCH on /$metadata and /$value; a proposal that does not suffer from the problem we are talking about.
This problem will always exist when doing a PATCH on /$value because there is no type information in the valueOnly serialization. From my perspective there are only two "fixes": change the valueOnly serialization to include type information (bad idea) or to not allow PATCH on /$value. 

I don't know what else to tell you or what else to present besides the simple solution I already proposed which is following best practices for REST APIs and just go with [RFC 7386 JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7386)